src/pcm.c

```diff
@@ -282,6 +282,11 @@ static unsigned int pcm_format_to_alsa(enum pcm_format format)
         return SNDRV_PCM_FORMAT_S32_LE;
     case PCM_FORMAT_S32_BE:
         return SNDRV_PCM_FORMAT_S32_BE;
+
+    case PCM_FORMAT_FLOAT_LE:
```

Comment:
Do you really want the new line here?

Reply:
Yes.

You would find that others big and little endian formats are also in groups and have a new line between each group.

Reply:
Ok, great. Thanks!